src/gui/folder.cpp

```diff
@@ -618,7 +618,8 @@ void Folder::slotWatchedPathChanged(const QString &path, ChangeReason reason)
                     spurious = false;
                 if (*pinState == PinState::OnlineOnly && record.isFile())
                     spurious = false;
-            }
+            } else
+                spurious = false;
```

Comment:
@m7913d Could you add braces for an `else` branch so it is easier to read this block?

Reply:
Done, it's just confusing braces aren't consistent in old source code.